pkg/provider/aws/resources/secret.go

```diff
@@ -10,13 +10,12 @@ import (
 type (
 	Secret struct {
 		Name          string
-		SecretName    string
 		ConstructsRef []core.AnnotationKey
 	}
 
 	SecretVersion struct {
-		SecretName    string
 		Secret        *Secret
+		DetectedPath  string
 		Path          string
```

Comment:
whats the difference between these?

Reply:
`DetectedPath` is set on creation and doesn't change -- it's intended more for the `Secrets` construct use-case. `Path` is configurable and defaults to `DetectedPath` if it's not set in klotho config. This helps with `Secrets` construct expansion where correlating a specific `Secret` resource back to a specific secret path can be awkward.